src/main/java/build/buildfarm/common/redis/RedisClient.java

```diff
@@ -131,7 +132,7 @@ public <T> T call(JedisContext<T> withJedis) throws IOException {
       // we are technically not at RESOURCE_EXHAUSTED, this is a
       // persistent state which can exist long past the error
       throw new IOException(Status.UNAVAILABLE.withCause(e).asRuntimeException());
-    } catch (JedisConnectionException e) {
+    } catch (JedisConnectionException | JedisClusterOperationException e) {
```

Comment:
JedisClusterOperationException is not guaranteed to have suppressions. It is also thrown in contexts that represent invalid arguments. If you're trying to get the UNAVAILABLE throw here for JedisClusterMaxAttemptsException, just catch that exception specifically and manufacture the exception, without any unwrapping.

Reply:
Updated